doc/source/reference/arrays.dtypes.rst

```diff
@@ -233,7 +233,7 @@ Array-protocol type strings (see :ref:`arrays.interface`)
    ``'m'``            timedelta
    ``'M'``            datetime
    ``'O'``            (Python) objects
-   ``'S'``, ``'a'``   (byte-)string
+   ``'S'``, ``'a'``   (byte-)string (not recommended)
```

Comment:
Probably worth unparenthesizing the (byte), and putting the word "string" after unicode on the line below, to stop people who are looking for string picking "S"?

Also, any way to link this to the admonition below?